Mathlib/LinearAlgebra/QuadraticForm/TensorProduct.lean

```diff
@@ -0,0 +1,88 @@
+/-
+Copyright (c) 2023 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+import Mathlib.LinearAlgebra.BilinearForm.TensorProduct
+import Mathlib.LinearAlgebra.QuadraticForm.Basic
+
+/-!
+# The quadratic form on a tensor product
+
+## Main definitions
+
+* `QuadraticForm.tensorDistrib (Q₁ ⊗ₜ Q₂)`: the quadratic form on `M₁ ⊗ M₂` constructed by applying
+  `Q₁` on `M₁` and `Q₂` on `M₂`. This construction is not available in characteristic two.
+
+-/
+
+universe uι uR uA uM₁ uM₂
+
+variable {ι : Type uι} {R : Type uR} {A : Type uA} {M₁ : Type uM₁} {M₂ : Type uM₂}
```

Comment:
```suggestion
universe uR uA uM₁ uM₂

variable {R : Type uR} {A : Type uA} {M₁ : Type uM₁} {M₂ : Type uM₂}
```
it doesn't seem iota is used in this file?

Reply:
Oops!